packages/webview_flutter/webview_flutter_platform_interface/lib/src/method_channel/webview_method_channel.dart

```diff
@@ -129,6 +129,18 @@ class MethodChannelWebViewPlatform implements WebViewPlatformController {
         .then((result) => result!);
   }
 
+  @override
+  Future<void> runJavaScript(String javaScriptString) async {
+    await _channel.invokeMethod<String>('runJavaScript', javaScriptString);
+  }
+
+  @override
+  Future<String> runJavaScriptForResult(String javaScriptString) {
```

Comment:
`For` sounds like the result is an argument; how about `runJavaScriptReturningResult`?